conf/alluxio-site.properties.template

```diff
@@ -13,71 +13,17 @@
 # Details about all configuration properties http://www.alluxio.org/documentation/en/Configuration-Settings.html
 
 # Common properties
-# alluxio.debug=false
-# alluxio.home=/mnt/alluxio_default_home
-# alluxio.test.mode=false
 # alluxio.underfs.address=${alluxio.work.dir}/underFSStorage
-# alluxio.web.threads=1
-
-# Integration properties
-# alluxio.integration.master.resource.cpu=1
-# alluxio.integration.master.resource.mem=1024MB
-# alluxio.integration.worker.resource.cpu=1
-# alluxio.integration.worker.resource.mem=1024MB
 
 # Security properties
 # alluxio.security.authentication.type=SIMPLE
-# alluxio.security.authorization.permission.enabled=true
-# alluxio.security.authorization.permission.supergroup=supergroup
```

Comment:
Can you please keep this `# alluxio.security.authorization.permission.enabled=true` because it's usually bundled with `SIMPLE` type.